.buildkite/ftr_configs.yml

```diff
@@ -24,6 +24,7 @@ disabled:
   - x-pack/test/upgrade/config.ts
   - test/functional/config.edge.js
   - x-pack/test/functional/config.edge.js
+  - x-pack/test/cloud_security_posture_functional/config.cloud.ts
```

Comment:
This means it will run on every pull request
I believe we agreed this is not necessary

Reply:
@kfirpeled, as @jbudz mentioned, this will not run because it is within the disabled block. The purpose of this configuration is to be executed on demand